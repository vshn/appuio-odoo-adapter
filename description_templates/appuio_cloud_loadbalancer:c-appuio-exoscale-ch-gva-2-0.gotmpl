Service Type LoadBalancer

{{template "_usage" .}}

Services of type LoadBalancer are not supported on "Exoscale - CH-GVA-2 0".