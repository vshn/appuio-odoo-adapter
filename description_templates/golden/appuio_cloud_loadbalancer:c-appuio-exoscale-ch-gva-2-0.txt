Service Type LoadBalancer

Qty: 87955674.09 UNIT-Minutes
Average Usage: 7.61 UNIT / Minute
Unit Price: CHF 0.0000007460 / UNIT / Minute

Services of type LoadBalancer are not supported on "Exoscale - CH-GVA-2 0".